assets/js/hooks/cell_editor/live_editor.js

```diff
@@ -389,11 +401,17 @@ export default class LiveEditor {
     // We dispatch escape event, but only if it is not consumed by any
     // registered handler in the editor, such as closing autocompletion
     // or escaping Vim insert mode
+    const ctrl = event.ctrlKey;
+    const alt = event.altKey;
+    const key = event.key;
 
-    if (event.key === "Escape") {
+    if (key === "Escape") {
       this.container.dispatchEvent(
         new CustomEvent("lb:editor_escape", { bubbles: true }),
       );
+      // On macOS, ctrl+alt+- becomes an em-dash, so we check for the code
+    } else if (event.code === "Minus" && ctrl && alt) {
```

Comment:
We have the global handler in session, so we should no longer need this one?

Reply:
Yeah, I removed that one